docs/installation.md

```diff
@@ -0,0 +1,136 @@
+# Installation
+
+## Table Of Contents
+
+- [Requirements](#requirements)
+- [Hyperconverged Cluster Operator (HCO) Installation](#hyperconverged-cluster-operator-hco-installation)
+- [Manually Installation](#manually-installation)
+- [Building](#building)
+- [Pause Operator](#pause-operator)
+
+## <a name="requirements"></a> Requirements
+
+SSP operator requires an OpenShift cluster in order to deliver complete functionality.
+
+Also it requires resource types and Custom Resource Definitions (CRDs) 
+for successful deployment across different Kubernetes distributions.
+
+### OpenShift
+
+| Resource Type / CRD                                                     | Needed By                                                                 |
+| ------------------------------------------------------------------------| --------------------------------------------------------------------------|
+| `dataimportcrons.cdi.kubevirt.io`                                       | `data-sources` operand (Kind `DataImportCron`)                            |
+| `datavolumes.cdi.kubevirt.io/v1beta1`                                   | `data-sources` operand (Kind `DataVolume` and `DataVolumeSource`)         |
+| `datasources.cdi.kubevirt.io/v1beta1`                                   | `data-sources` operand (Kind `DataSource`)                                |
+| `prometheusrules.monitoring.coreos.com`                                 | `metrics` operand (Kind `PrometheusRule`)                                 |
+| `template.openshift.io/v1`                                              | `common-templates` operand (Kind `Template`)                              |
+| `virtualmachineclusterinstancetypes.instancetypes.kubevirt.io/v1alpha2` | `common-instancetypes` operand (Kind `VirtualMachineClusterInstancetype`) |
```

Comment:
nit - These `instancetype.kubevirt.io` resources should be `instancetype.kubevirt.io/v1beta1`, happy to fix in a follow up or for you to do it here if you need to amend this PR.

Reply:
Done.